src/wp-includes/class-wp-block-bindings-registry.php

```diff
@@ -0,0 +1,194 @@
+<?php
+/**
+ * Block Bindings API: WP_Block_Bindings_Registry class.
+ *
+ * Supports overriding content in blocks by connecting them to different sources.
+ *
+ * @package WordPress
+ * @subpackage Block Bindings
+ * @since 6.5.0
+ */
+
+/**
+ * Core class used for interacting with block binding sources.
+ *
+ *  @since 6.5.0
+ */
+final class WP_Block_Bindings_Registry {
+
+	/**
+	 * Holds the registered block bindings sources, keyed by source identifier.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @var array
```

Comment:
The line break is not needed, similar to the property below.

Reply:
@gziolo ☝️

Reply:
I'll include that change in the next batch of changes planned: https://github.com/WordPress/wordpress-develop/pull/5966.